src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+              <a href="/tags/choose">choose</a> the companies we patronize, to
+              encourage positive trends
+            </li>
+            <li>
+              <a href="/tags/inspire">inspire</a> others to understand the
+              gravity of the problems we're solving
+            </li>
+          </ul>
+
+          <p>
+            Relying on one step of the plan or squabbling about which ones are
+            good or bad will lead to confusion and inaction. We definitely want
+            people to be offsetting their footprint, so long as that the
+            messaging is clear: offsetting is not a free pass top continue
+            living a polluting life, but it is a valuable tool to balance your
+            remaining footprint whilst you make drastic reductions month by
+            month.{' '}
+          </p>
+
+          <p>
+            If all individuals, companies, and governments can start with reduce
+            their emissions sharply, and we all increase our funding for these
+            projects by offsetting whatever footprints remain, then eventually
+            we'll need less and less offsetting to be truly carbon neutral, then
+            shortly after be climate positive. We've got to sort this out now,
+            so if you wanna sign up for an account with an offset program today,
+            then start figuring out where to make the biggest changes in your
+            life tomorrow, that's fine. The important thing is to do both, and
+            quickly.{' '}
+          </p>
+
+          <h2 id="checkforstandards">Check for Standards</h2>
+
+          <p>
+            An offsetting organization must clearly display the independent
+            auditing or standards it uses, like{' '}
+            <a href="https://www.goldstandard.org/">Gold Standard</a>,{' '}
+            <a href="https://verra.org/">Verified Carbon Standard</a>,{' '}
+            <a href="https://www.green-e.org/programs/climate">
+              Green-e® Climate Standard
+            </a>
+            , or the United Nations{' '}
+            <a href="https://offset.climateneutralnow.org/">
+              Certified Emission Reductions
+            </a>{' '}
+            (CERs). Projects that meet the criteria for these standards are
+            rigorously audited by external auditors, usually from non-profit
+            organizations and charities like the WWF, who make sure the people
+            on the ground are indeed planting the trees, installing the solar
+            panels, etc.
+          </p>
+
+          <p>
+            Organizations like <a href="https://offset.earth/">Offset Earth</a>{' '}
+            and{' '}
+            <a href="https://co2.myclimate.org/en/offset_further_emissions">
+              myclimate
+            </a>{' '}
+            do this homework for you and only pick certified projects, so you
+            can be confident your money is getting to the best place.{' '}
+          </p>
```

Comment:
Add disclaimer here about orgs only putting a fraction of what they could be into offsetting and that's **really** hard to figure out, but transparency is key and we've done the homework for you.  Offset earth puts 97%, etc etc..